packages/utilities/psammead-assets/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.12.2 | [PR#3020](https://github.com/bbc/psammead/pull/3020) Add SVG for Learning English brand logo |
```

Comment:
This should be a minor update since it's adding a new export.
```suggestion
| 2.13.0 | [PR#3020](https://github.com/bbc/psammead/pull/3020) Add SVG for Learning English brand logo |
```